### ISSUE TITLE
Need help understanding and improving

### ISSUE THREAD

**Issue by divinediu** [ellisdg/3DUnetCNN#214]
Hi, I am using this to detect tumors in CT images. 

I have 75 CT images with different XxYxZ dimensions, each with a segmentation file with only one label. The segmentation of the tumor.
<File_1, Tumor_in_file_1>

How many labels will I have? The Net didn't take 1(the tumor) as input. So I gave 2.

config["batch_size"] = 6
config["validation_batch_size"] = 4

I am getting this after running for about 100 epochs

 24s/step - loss: -0.0717 - dice_coefficient: 0.0717 - val_loss: -0.1015 - val_dice_coefficient: 0.1015

What is the difference between dice coefficient and val_dice_coefficient?

How do I improve this?

**Comment by nysuka** [comment]
to train 1 modality and 1 segmentation label, there is just 1 label, just need to change in train_isensee2017.py

`config["labels"] = (1, )`
`config["all_modalities"] = ["name of ct file"]`

the tumor is the truth file.

loss is negative of dice_coefficient for the training data set.
val_loss is negative of the the val_dice_coefficient for the validation data set.
loss metric is negative since the optimization is a minimization.
